### PR TITLE
German spoken language fixed.

### PR DESCRIPTION
**Pull request by rcbebo82**
Spoken numbers fixed:
100 -> "ein", "hundert"
1000 -> "ein", "tausend"
999 -> "9", "hundert", "99"

Timers:
60 seconds -> "eine", "minute"
3600 seconds -> "eine", "stunde"
120 seconds -> "2", "minuten"

Sound files needed to be generated because new words are used.

### CODE REVIEW

**Comment by rcbebo82** [comment]
Got it with the SmartGit Client.
Now the new code file and the new python file for generating sound files!

File: radio/util/tts_de.py [hunk 1]
```diff
@@ -8,11 +8,14 @@
...
+for i in range(100):
     systemSounds.append((str(i), filename(PROMPT_SYSTEM_BASE + i)))
-systemSounds.append(("tausend", filename(PROMPT_SYSTEM_BASE + 101)))
-for i, s in enumerate(["comma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden"]):
-    systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 102 + i)))
+systemSounds.append(("ein", filename(PROMPT_SYSTEM_BASE + 100)))
+systemSounds.append(("eine", filename(PROMPT_SYSTEM_BASE + 101)))
+systemSounds.append(("hundert", filename(PROMPT_SYSTEM_BASE + 102)))
+systemSounds.append(("tausend", filename(PROMPT_SYSTEM_BASE + 103)))
+for i, s in enumerate(["komma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden", "stunde", "stunden"]):
```
Comment (schwabe) [inline radio/util/tts_de.py]: did you check that komma really sounds better than comma? The mispelling are sometimes to get better pronounciation.

Comment (rcbebo82) [inline radio/util/tts_de.py]: It is pronounced correctly. With pico2wave there is even now difference recognizable.

**Comment by bsongis** [comment]
I will let german speaking people press the button for that one, ok?

**Comment by 3djc** [comment]
I'll do some more test tomorrow, I need to generate a audio pack first

**Comment by 3djc** [comment]
I believe there is an issue when you want to play values at 0, you just have the unit and no 'zero' value

**Comment by rcbebo82** [comment]
I will recheck that. Could be possible that i didnt test this!

**Comment by rcbebo82** [comment]
Issue confirmed, working on the code again.

**Comment by rcbebo82** [comment]
Zero works now, but im stuck at the precesion. Could anyone explain me how a telemetry source with precesion 0.00 is handled. 
Concretly i would to know how the parameter precesion on a telemtry source is set in code:
https://github.com/opentx/opentx/blob/master/radio/src/translations/tts_de.cpp#L103
So which "mode" is set by precesion:
0 -> 0?
0.0 -> 1?
0.00 -> 2?

**Comment by 3djc** [comment]
Let's say your value is 1024, 
PREC 0 is 1024
PREC1 is 102,4
PREC2 is 10,24

**Comment by rcbebo82** [comment]
Hi, that was already clear to me. But the value "mode" controls which precesion is used. 
German language is really a nightmare. A small example:

This works already:
Precesion=0, Unit=Volt:, Value=1
"ein" "Volt"
Precesion=0, Unit=Volt:, Value=2
2 "Volt"

Here it starts to broken down:
Precesion=1, Unit=Volt:, Value=11
Should be spoken: 1 "komma" 1 "Volt"
Is spoken: "ein" "komma" "ein" "Volt"

Works:
Precesion=1, Unit=Volt:, Value=22
2 "komma" 2 "Volt"

Defect:
Precesion=2, Unit=Volt:, Value=111
"hundert" "komma" "hundert" "Volt"

Precesion=2, Unit=Volt:, Value=222
2 "komma" 2 "Volt"

**Comment by rcbebo82** [comment]
TRACE("DEBUG MODE : %d", mode);
Thx will try this!

**Comment by rcbebo82** [comment]
Is it possible that precesion 0.00 does not work anymore also in english. If i choose as source VFAS with a value of 4,85 Volt and telemetry sensor set to precesion 0.00 i get as number 49 instead of 485.
Or is the spoken part limited to precesion 0.0?

**Comment by 3djc** [comment]
Not sure what you are refereing too, since even with 0.00 precisiion (defaut for VFAS sensor), the 'play value' function uses 0.0 precision

**Comment by rcbebo82** [comment]
So play value uses in this context every time 0.0 precesion? This means if i switch to precesion 0.00 in a telemetry source every time 0.0 is used?

**Comment by kilrah** [comment]
Yes, voice callouts have always only been 1 decimal.
Makes little sense calling out 2 in most cases.

**Comment by 3djc** [comment]
Tested it (with my limited german),, sounds ok

**Pull request merged**